test/benchmark/avro-upsert/metric-verifier/wait_metric

```diff
@@ -10,32 +10,63 @@
 # by the Apache License, Version 2.0.
 
 import argparse
+import dateparser
+import datetime
 import sys
 import time
+import urllib
 
 import prometheus_api_client
+import prometheus_api_client.utils
 
 def run(args: argparse.Namespace) -> int:
-    """Run the generator, inserting args.num_records number of messages."""
+    """Wait for the query to settle or timeout and then dump ingest metrics."""
 
-    prom = prometheus_api_client.PrometheusConnect(f'http://{args.host}:{args.port}')
+    try:
+        start = dateparser.parse('30 seconds ago')
+        return wait_metric(args)
+    finally:
+        print_dashboard_url(args, start)
+
+
+def print_dashboard_url(args: argparse.Namespace, start: datetime.datetime):
+    """Print the URL to see the metrics for this test"""
 
+    # Create parameters to see a dashboard with the metrics from this benchmark run
+    # Convert start and end to timestamps, with a little padding to make the dashboard look nicer
+    # Grafana expects timestamps with milliseconds
+    path = '/d/materialize-overview/materialize-overview'
+    query = urllib.parse.urlencode( {
+              "from": round(start.timestamp()) * 1000,
+                "to": round(dateparser.parse('in 30 seconds').timestamp()) * 1000,
```

Comment:
its not clear to me how to `"to"` timestamp depends on the end timestamp - I'm probably missing something and would love to know what

Reply:
Are you looking at an older version of the code? The current code calculates `to` without using an `end` timestamp.

Also, if we didn't supply a value for `to`, then the dashboard would default to an interval that may include other benchmarks.

Reply:
Oh, I see -- the comment mentions `end`. Will fix up!